audit/projects/k8s-infra-e2e-boskos-gpu-01/buckets/kubernetes-staging-a0b6b040d4/cors.txt

```diff
@@ -0,0 +1 @@
+gs://kubernetes-staging-a0b6b040d4/ has no CORS configuration.
```

Comment:
These buckets get created by kube-up
https://github.com/kubernetes/kubernetes/blob/703e2bd2ebae1cc7737c12344a9e6fb13a6019a9/cluster/gce/util.sh#L306-L316

https://github.com/kubernetes/kubernetes/blob/703e2bd2ebae1cc7737c12344a9e6fb13a6019a9/cluster/gce/util.sh#L341-L347

Since they're based on project hash, they should be consistent, and we shouldn't expect too many of these per e2e project